server/sonar-web/src/main/js/components/shared/TestStatusIcon.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import * as classNames from 'classnames';
+
+interface Props {
+  className?: string;
+  status: string;
+}
+
+export default function TestStatusIcon({ className, status }: Props) {
```

Comment:
Maybe this icon can be replaced by the usage of `CheckIcon` and `ClearIcon`. I guess it depends on what the possible statuses are.

Reply:
Those icons are different (no circle outline). I'd need to use `AlertSuccessIcon` and `AlertErrorIcon` from your PR about webhooks.

Reply:
Ah yes true, it's on our PR...